src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1844,8 +1846,11 @@ public void restartServer(boolean buildContainer) throws PluginExecutionExceptio
         libertyCreate();
         // Skip installing features on container during restart, since the Dockerfile
         // should have 'RUN features.sh'
-        if (!container) {
+        // Also skip install feature on restart if config parameter specified.
+        if (!container && !skipInstallFeature) {
             libertyInstallFeature();
+        } else {
+            info("Skipping liberty:install-feature");
```

Comment:
```suggestion
            info("Skipping Liberty install feature.");
```

Maven/Gradle-neutral msg?

Reply:
I modified the message. Can you approve?